KubeArmor/monitor/systemMonitor.go

```diff
@@ -406,6 +403,9 @@ func (mon *SystemMonitor) InitBPF() error {
 
 // DestroySystemMonitor Function
 func (mon *SystemMonitor) DestroySystemMonitor() error {
+	mon.Status = false
+	time.Sleep(1 * time.Second)
```

Comment:
```suggestion
```
This sleep wont help since the sleep in SystemMonitor is of 10 sec. Better to remove the sleep completely.